wpeverest/everest-forms
# Fixed - Password Validation logic added

### All Submissions:






### Changes proposed in this Pull Request:



Show hide logic added for Password Validation PR - https://github.com/wpeverest/everest-forms-pro/pull/338

### How to test the changes in this Pull Request:

1. Create Form with Password Field.
2. Enable password strength meter.
3. Check Strength Text
4. Preview the form
5. Verify the changes in Password field.

### Types of changes:




### Other information:




### Changelog entry
> Fixed - Password Validation show hide logic added
